core/ibft_test.go

```diff
@@ -2283,3 +2283,41 @@ func TestIBFT_ExtendRoundTimer(t *testing.T) {
 	// Make sure the round timeout was extended
 	assert.Equal(t, additionalTimeout, i.additionalTimeout)
 }
+
+func Test_getRoundTimeout(t *testing.T) {
+	type args struct {
+		baseRoundTimeout  time.Duration
+		additionalTimeout time.Duration
+		round             uint64
+	}
+	tests := []struct {
+		name string
+		args args
+		want time.Duration
+	}{
+		{
+			name: "first round duration",
+			args: args{
+				baseRoundTimeout:  time.Second,
+				additionalTimeout: time.Second,
+				round:             0,
+			},
+			want: time.Second * 2,
```

Comment:
just for a future task, could we remove using time in tests at all? use timeout events from consensus, or round numbers?

Reply:
but this is just a unit test for the timeout calculation logic

Reply:
yes, although if no timeouts, sleeps, etc we'd be able to run times more tests per minute.

Reply:
I mean this test is for the function that calculates a round duration. Don't really get what sleeps are you talking about.